app/models/job_posting.rb

```diff
@@ -4,9 +4,13 @@ class JobPosting < ActiveRecord::Base
   validates :description, presence: true
   validates :how_to_apply, presence: true
 
+  include Rakismet::Model
+  rakismet_attrs :comment_type => 'job-post', :content => :description,
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.